build_scripts/build_standalone_binaries.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Script builds the standalone applications
+
+SCRIPT_DIR=`pwd`/`dirname "$0"`
+BUILD_DIR=$SCRIPT_DIR/build
+SYSROOT_DIR=$BUILD_DIR/pluto-custom.sysroot
+SRC_DIR=$SCRIPT_DIR/..
+
+echo "Building main application..."
+mkdir -p $BUILD_DIR/cmake-build
+
+if [[ ! -d $SYSROOT_DIR ]]
+then
+  ./create_sysroot.sh
+else
+  echo "Sysroot is present."
+fi
+
+cd $BUILD_DIR/cmake-build || exit 1
+export PLUTO_SYSROOT_DIR=$SYSROOT_DIR
+cmake -DCMAKE_TOOLCHAIN_FILE=$SRC_DIR/CmakeArmToolchain.cmake $SRC_DIR
+make basestation
+make client
+make client-calib
```

Comment:
Each of the make commands should fail with `|| exit 1`. Currently, the script is deemed a successful, even if one or more of the builds is unsuccessful.